packages/sdk/examples/simple/index.ts

```diff
@@ -0,0 +1,113 @@
+import { AssetSymbol, ChainKey } from '@moonbeam-network/xcm-config';
+import { init, toDecimal } from '@moonbeam-network/xcm-sdk';
+import { Keyring } from '@polkadot/api';
+import { ethers } from 'ethers';
+import { setTimeout } from 'timers/promises';
+
+/**
+ * Add your moonbeam address and private key
+ */
+const moonbeamAddress = '';
```

Comment:
Maybe we can drop the addresses here, as we can get them from the private keys by doing `signer.address` and `polkadotAccount.address`

Reply:
I had some issues with that so I decided to do it dum way. Maybe we can change this in the furure.